### ISSUE TITLE
[English]: Content Delivery Network

### ISSUE THREAD

**Issue by mdarmandev** [cncf/glossary#532]
### Details (optional)

Optional details

**Comment by karnatisrinivas** [comment]
Hello @mdarmandev , Can I know the context of issue please!

**Comment by mdarmandev** [comment]
i want to contribute in the glossary !

**Comment by CathPag** [comment]
Here you go, @mdarmandev. Please make sure to read the[ style guide](https://glossary.cncf.io/style-guide/) before getting started. 

There is an update that hasn't been merged yet but is really important so I'm pasting it in here:

### Quality is paramount 
If merged, your submission will be the official CNCF definition for that term (until someone else improves it). Creating a term that meets the CNCF’s high standards can’t be rushed — quality takes time and effort.

Do your research: Even if you are confident you know the term, verify you go it right. We often use terms in organizations a certain way that may not reflect the full picture. When you do your research, especially when you’re not 100% familiar with the term, use multiple resources. Many definitions are one-sided, especially if provided by a vendor. The Glossary must have vendor-neutral, globally accepted definitions.

Don’t plagiarize. The same rules apply to Glossary as to any other serious publication. Don’t copy and paste other people’s work unless you are quoting and contributing it to them. If you like a particular section of a definition, paraphrase it (describe it in your own words).

**Comment by CathPag** [comment]
> Hello @mdarmandev , Can I know the context of issue please!

So all these issues with just one term in the subject line are new term suggestions. If you want to contribute, pick a term you feel comfortable with, asked to be assigned to it, and work on it. But before you do any of that, please read the [how to contribute](https://glossary.cncf.io/contribute/) and [style guide.](https://glossary.cncf.io/style-guide/)

**Comment by mdarmandev** [comment]
@CathPag Can I open issue for new term suggestions while researching about this.

**Comment by CathPag** [comment]
> @CathPag Can I open issue for new term suggestions while researching about this.

Yes, @mdarmandev, if you want to work on a term that doesn't exist or doesn't have an issue yet, you can submit a new issue. The maintainers will assess if it should be part of the Glossary and, if so, assign it to you.

**Comment by akshatcoder-hash** [comment]
Hey @CathPag ! I would love to work on this issue. Could you please assign this to me?

**Comment by CathPag** [comment]
The maintainers reviewed this term and came to the conclusion that it doesn't meet the requirements to be added to the glossary. I'll go ahead and close this issue.
